atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/SweepBatchAccumulator.java

```diff
@@ -33,6 +33,7 @@ class SweepBatchAccumulator {
     private final int batchSizeThreshold;
 
     private long progressTimestamp;
+    private long lastSeenCommitTimestamp = 0L;
     private long entriesRead = 0;
```

Comment:
why does this have to be 0 and not -1

Reply:
Just in case because if you try to persist -1, it will explode since it encodes it as unsigned